src/Scene/LayersConfiguration.js

```diff
@@ -18,6 +18,12 @@ function LayersConfiguration() {
     this.geometryLayers = [];
     this.colorLayers = [];
     this.elevationLayers = [];
+    this.lightingLayers = [];
```

Comment:
Not really.
We can simply acknowledge that the current implementation is not perfect to implement these features. And then fix these issues in PR 239 (or later).